packages/oid4vci-demo-frontend/public/locales/development/en/translation.json

```diff
@@ -4,21 +4,19 @@
   "credentials_right_pane_bottom_paragraph": "Scan this QR code using your wallet's QR\nscanner to accept and store your credential.",
   "credentials_right_pane_top_paragraph": "Let's add a {{credentialName}} Credential\nto your wallet.",
   "credentials_right_pane_top_title": "Secure your {{credentialName}} Credential",
-  "credentials_success_right_pane_back_button_caption": "Back to credentials",
```

Comment:
No longer in use?

Reply:
there a lot not in use anymore think we need to look at this later